mkdocs.yml

```diff
@@ -0,0 +1,15 @@
+site_name: ddev Documentation
+edit_uri: edit/master/docs/
+copyright: DRUD Technology, LLC
+theme: readthedocs
```

Comment:
We definitely need a followup issue to have our own theme, or at least a different one. I assume we have canned options out there. Is it worth exploring any of them at this time?